src/App.jsx

```diff
@@ -1,15 +1,23 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css'
-import TodoList from './TodoList';
+import TodoList, { initialTodoList } from './TodoList';
 import AddTodoForm from './AddTodoForm';
 
 function App() {
+    const [newTodoList, setTodoList] = useState( initialTodoList );
```

Comment:
when we are using useState by convention we use the name for the function state setter in useState with the word 'new' in the beginning and state name. [todo, newTodo], [word, newWord] etc.

Reply:
Got it, thank you!